Packs/CortexResponseAndRemediation/ReleaseNotes/1_1_51.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Office process creates a scheduled task via file access
+
+Added 'Continue on error' in the "Check the initiator process reputation" task.
```

Comment:
What about 'Added 'Continue on error' option in the ""Check the initiator process reputation" task.  (adding the word 'option' - if that's what happened)